source/_integrations/yale.markdown

```diff
@@ -0,0 +1,163 @@
+---
+title: Yale
+description: Instructions on how to integrate your Yale devices into Home Assistant.
+ha_category:
+  - Binary sensor
+  - Button
+  - Camera
+  - Doorbell
+  - Event
+  - Lock
+  - Sensor
+ha_release: 2024.9
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@bdraco'
+ha_domain: yale
+ha_dhcp: true
+ha_platforms:
+  - binary_sensor
+  - button
+  - camera
+  - diagnostics
+  - event
+  - lock
+  - sensor
+ha_integration_type: integration
+---
+
+The **Yale** integration allows you to integrate your [Yale](https://www.assaabloy.com/vn/en/solutions/products/yale) and some Yale Access devices in Home Assistant.
+
+The login credentials used to authenticate the integration are the same as the ones used in the Yale Home app.
+
+{% include integrations/config_flow.md %}
+
+## Known working devices
+
+| Device                            | Requires [Connect Bridge](https://www.yalehome.com/vn/en/products/yale-smart-door-lock/smart-accessories/yale-connect-wi-fi-bridge) or Doorbell |
+| --------------------------------- | ------------------------------------|
+| Yale Wi-Fi Smart Lock (Gen 4) | no |
+| Yale Smart Lock Pro (Gen 3) | yes |
+| Yale Smart Lock (Gen 2) | yes |
+| Yale Smart Lock (Gen 1) | no |
+| Yale Doorbell Cam (Gen 1, Gen2) | no |
+| Yale View | no |
```

Comment:
```suggestion
```

Reply:
These locks are only available in the North American market (ie Yale Access, not Yale / Yale Global / Yale Home )